dgraphtest/acl.go

```diff
@@ -79,7 +79,7 @@ func (hc *HTTPClient) GetCurrentUser() (string, error) {
 	params := GraphQLParams{Query: query}
 	resp, err := hc.RunGraphqlQuery(params, true)
 	if err != nil {
-		return "", errors.Wrapf(err, "received response: %v", string(resp))
+		return string(resp), errors.Wrapf(err, "received response: %v", string(resp))
```

Comment:
What is the function actually responding? Why do u need resp here?

Reply:
yeah, I agree with Harshil here. If we are returning an error, all the details should be in the error and the first return value should be zero value. Let me fix it.

Reply:
fixed it, not sure why GitHub is not showing my new commit.